src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -150,7 +150,7 @@ def _process_input(conf: Conf):
             os.link(cfn, dst)
 
             # save meta file with relevant object_info
-            loader_helper.dump_json_to_file(metafile, loader_helper.generate_import_dir_meta(obj_info, genome_info))
+            loader_helper.create_meta_file(conf.output_dir, upa_format, obj_info, genome_info)
```

Comment:
This should return the meta file location since it's used below, rather than having it declared above

Reply:
This step is in `if` condition and can be skipped.

Reply:
I do think it makes sense to return the metadata file for the helper function. But we cannot use it here.

Reply:
Maybe there should be a function for just getting the file location then?

What happens if the file isn't created and the function keeps going?

Reply:
> Maybe there should be a function for just getting the file location then?

👍 

> What happens if the file isn't created and the function keeps going?

It won't. The file existence and content are checked by the if condition.

Reply:
:+1: